src/charm.py

```diff
@@ -361,12 +361,18 @@ def _render_conf_file(self) -> str:
         login_ui_url = self._get_login_ui_endpoint_info("login_url")
         mappers = self._get_claims_mappers()
         cookie_secrets = self._get_secret()
+
+        allowed_return_urls = []
+        if self._public_url:
+            allowed_return_urls.append(domain_url(self._public_url))
```

Comment:
```suggestion
            allowed_return_urls.append(urlparse(self.public_url)._replace(path="*", params="", query="", fragment="").geturl())
```


I would put it inline, jumping across module for a function that is a little wrapper of a built-in function doesn't seem that intuitive